src/test/java/domain/bet/BetTest.java

```diff
@@ -0,0 +1,62 @@
+package domain.bet;
+
+import domain.result.Outcome;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class BetTest {
+
+    @DisplayName("최초의 배팅금액이 0이하이면 예외를 던진다.")
+    @ParameterizedTest
+    @ValueSource(ints = {-100, 0, -1})
+    void constructBetTest(final int money) {
+        assertThrows(IllegalArgumentException.class,
+                () -> new Bet(money))
+                .getMessage().equals("[ERROR]: 배팅금액은 0원 초과여야 합니다.");
+    }
+
+    @DisplayName("블랙잭으로 이겼을 때 배팅금액의 1.5배를 받는다.")
+    @Test
+    void calculateBetByOutcomeWinWithBlackJackTest() {
+        Bet bet = new Bet(1000);
+        bet.calculateBetByOutcome(Outcome.WIN, true);
+        int expectedMoney = 1500;
+
+        assertEquals(expectedMoney, bet.getMoney());
+    }
```

Comment:
```suggestion
    void calculateBetByOutcomeWinWithBlackJackTest() {
        int bettingMoney = 1000;
        Bet bet = new Bet(bettingMoney);
        double expectedMoney = bettingMoney * 1.5;

        bet.calculateBetByOutcome(Outcome.WIN, true);
        
        assertEquals(expectedMoney, bet.getMoney());
    }
```

expectedMoney를 직접 작성하기 보다는 게임 규칙을 활용해서 선언할수 있을 것 같아요.

Reply:
감사합니다 ㅎㅎ